test/data/test_builtin_datasets.py

```diff
@@ -146,19 +48,12 @@ def test_raw_text_name_property(self, info):
     def test_raw_text_classification(self, info):
         dataset_name = info['dataset_name']
         split = info['split']
-
-        if dataset_name == 'WMT14':
```

Comment:
we probably want to keep around testing for raw experimental datasets unless we also decide to either remove it or graduate it to make it part of torchtext datasets.